src/mono/mono/metadata/marshal.c

```diff
@@ -6755,3 +6770,34 @@ get_marshal_cb (void)
 	}
 	return &marshal_cb;
 }
+
+/**
+ * mono_method_has_unmanaged_callers_only_attribute:
+ *
+ * Returns \c TRUE if \p method has the \c UnmanagedCallersOnlyAttribute
+ */
+gboolean
+mono_method_has_unmanaged_callers_only_attribute (MonoMethod *method)
+{
+	/* TODO: if this is slow and called repeatedly for the same method, add a bit to MonoMethod that is true if the
```

Comment:
Just a reminder to remove this comment as you moved the invocation to method_to_ir() so it (probably) doesn't apply anymore